packages/workbox-routing/test/sw/router.js

```diff
@@ -73,4 +49,32 @@ describe(`Test of the Router class`, function() {
     expect(router._routes.get('GET')).to.have.members([getRoute1]);
     expect(router._routes.get('PUT')).to.have.members([putRoute1]);
   });
+
+  it(`should call self.addEventListener('fetch') when addFetchListener() is called`, function() {
+    const stub = sinon.stub(self.__proto__.__proto__.__proto__, 'addEventListener');
```

Comment:
This seems super strange :) I'm curious about the proto depth here, but imagine there's a good reason.